lib/onebox/helpers.rb

```diff
@@ -178,6 +178,10 @@ def self.normalize_url_for_output(url)
       url.gsub!("'", "&apos;")
       url.gsub!('"', "&quot;")
       url.gsub!(/[^\w\-`.~:\/?#\[\]@!$&'\(\)*+,;=%\p{M}’]/, "")
+
+      parsed = Addressable::URI.parse(url)
+      return "" unless parsed.host && parsed
```

Comment:
```suggestion
      return "" unless parsed.host
```

Are there any cases where `parsed` could be falsy? Other than `url` being nil, but that won't happen in this case.